image.go

```diff
@@ -20,6 +20,8 @@ type Image interface {
 	Target() ocispec.Descriptor
 	// Unpack unpacks the image's content into a snapshot
 	Unpack(context.Context, string) error
+	// Adding a function to access internal structure
+	Info() images.Image
```

Comment:
I was just following the way it was used in container.go. Happy to make the change.